straxen/plugins/peaklet_processing.py

```diff
@@ -557,35 +558,37 @@ class MergedS2s(strax.OverlapWindowPlugin):
     depends_on = ('peaklets', 'peaklet_classification')
     data_kind = 'merged_s2s'
     provides = 'merged_s2s'
+    __version__ = '0.1.0'
 
     def infer_dtype(self):
         return strax.unpack_dtype(self.deps['peaklets'].dtype_for('peaklets'))
 
     def get_window_size(self):
-        return 5 * (self.config['s2_merge_max_gap']
+        return 5 * (int(self.config['s2_merge_gap_thresholds'][0][1])
                     + self.config['s2_merge_max_duration'])
 
     def compute(self, peaklets):
-        if not len(peaklets):
+        if len(peaklets) <= 1:
```

Comment:
out of curiosity, for 1 peaklet, we don't have to return a mergedS2 right as the data is from the peaklets already?

Reply:
Right, not sure if there's a difference between return empty list and None.

Reply:
Giving the instruction to overwrite a single peak with the same single peak is not really needed unless I misunderstand